npm/wizard/src/templates/vue/vueWebpackFile.ts

```diff
@@ -0,0 +1,41 @@
+import path from 'path'
+import { Template } from '../Template'
+import { findWebpackConfig } from '../templateUtils'
+
+export const VueWebpackTemplate: Template<{ webpackConfigPath: string }> = {
+  message:
+    'It looks like you have custom `webpack.config.js`. We can use it to bundle the components for testing.',
+  getExampleUrl: () => {
+    return 'https://github.com/cypress-io/cypress/tree/develop/npm/vue/examples/cli'
+  },
+  recommendedComponentFolder: 'cypress/component',
+  getPluginsCode: (payload, { cypressProjectRoot }) => {
+    const includeWarnComment = !payload
+    const webpackConfigPath = payload
+      ? path.relative(cypressProjectRoot, payload.webpackConfigPath)
+      : './webpack.config.js'
+
+    return [
+      'const {',
```

Comment:
Should we use [ejs](https://ejs.co/) templates here? 
It would feel simpler to read and to maintain.

Reply:
not sure about this, because this should never change. But maybe it makes sense, we discussed this already